src/views/document/RouteView.vue

```diff
@@ -180,6 +185,27 @@ export default {
 
       return result;
     },
+    showMissingHistoryBanner() {
+      const doc = this.document;
+      const activities = doc.activities ?? [];
+      const history_worth_activities = [
```

Comment:
Please use camelCase notation for variables. We have some properties with underscores, but that's because they come directly from the api.